src/browser/browser.js

```diff
@@ -95,6 +95,9 @@ module.exports = class Browser {
                             this.state.testXReqId
                         }${X_REQUEST_ID_DELIMITER}${crypto.randomUUID()}`;
                     }
+                    if (!req.headers["traceparent"] && this.state.traceparent) {
```

Comment:
Looks like we should write something about this new header field in documentation.